src/test/regress/expected/multi_task_assignment_policy_0.out

```diff
@@ -176,3 +176,84 @@ RESET client_min_messages;
+   Task Count: 1
+   Tasks Shown: All
+   ->  Task
+         Node: host=localhost port=57638 dbname=regression
+         ->  Seq Scan on task_assignment_reference_table_880000 task_assignment_reference_table  (cost=0.00..35.50 rows=2550 width=4)
+(6 rows)
+
+SET citus.task_assignment_policy TO 'first-replica';
+EXPLAIN SELECT * FROM task_assignment_reference_table;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (Citus Router)  (cost=0.00..0.00 rows=0 width=0)
+   Task Count: 1
+   Tasks Shown: All
+   ->  Task
+         Node: host=localhost port=57638 dbname=regression
+         ->  Seq Scan on task_assignment_reference_table_880000 task_assignment_reference_table  (cost=0.00..35.50 rows=2550 width=4)
+(6 rows)
+
+EXPLAIN SELECT * FROM task_assignment_reference_table;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (Citus Router)  (cost=0.00..0.00 rows=0 width=0)
+   Task Count: 1
+   Tasks Shown: All
+   ->  Task
+         Node: host=localhost port=57638 dbname=regression
+         ->  Seq Scan on task_assignment_reference_table_880000 task_assignment_reference_table  (cost=0.00..35.50 rows=2550 width=4)
+(6 rows)
+
+SET citus.task_assignment_policy TO 'round-robin';
+EXPLAIN SELECT * FROM task_assignment_reference_table;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (Citus Router)  (cost=0.00..0.00 rows=0 width=0)
+   Task Count: 1
+   Tasks Shown: All
+   ->  Task
+         Node: host=localhost port=57637 dbname=regression
+         ->  Seq Scan on task_assignment_reference_table_880000 task_assignment_reference_table  (cost=0.00..35.50 rows=2550 width=4)
+(6 rows)
+
+EXPLAIN SELECT * FROM task_assignment_reference_table;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (Citus Router)  (cost=0.00..0.00 rows=0 width=0)
+   Task Count: 1
+   Tasks Shown: All
+   ->  Task
+         Node: host=localhost port=57638 dbname=regression
+         ->  Seq Scan on task_assignment_reference_table_880000 task_assignment_reference_table  (cost=0.00..35.50 rows=2550 width=4)
```

Comment:
we always disable costs when using EXPLAIN in regression tests, since the cost estimate may change over time / different postgres versions